docs/source/faq/index.rst

```diff
@@ -0,0 +1,45 @@
+FAQ
+===
+
+I'm new. Where to start?
+------------------------
+
+Hello, newcomer! You are very welcome :) \
```

Comment:
What is the purpose of backslashes?
Paragraphs can be split using a blank line in rst source.